assets/questions/strobogrammatic.js

```diff
@@ -0,0 +1,88 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for strobogrammatic number.
+ */
+
+globalData.questions['strobogrammatic'] = {  // eslint-disable-line dot-notation
+  title: 'strobogrammatic',
+  starterCode: {
+    python:
+`def strobogrammatic(length):
+    return []
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def firstDigitShouldNotBeZero(cls, length):
+        if length == 2:
+           return [["00","11","69","88","96"]]
+
+    @classmethod
+    def forgetZero(cls, length):
+        if length == 1:
+           return [["1","8"]]
+`
+  },
+  tasks: [{
+    instructions: [[ 
+      'Implement a function which takes a number N as input and generates a list of ',
+      'numbers which has length N and has rotational symmetry ',
```

Comment:
Implement a function strobogrammatic that takes a number n as input and generates a list of n-digit numbers which look the same if you rotate them 180 degrees (look at them upside-down). 

For example, all the 2-digit strobogrammatic numbers are [11, 69, 88, 96].